docs/cmdline-opts/mptcp.md

```diff
@@ -0,0 +1,37 @@
+---
+c: Copyright (C) Dorian Craps, <dorian.craps@student.vinci.be>
+SPDX-License-Identifier: curl
+Long: mptcp
+Added: 8.7.0
+Help: Enable Multipath TCP (MPTCP)
+Category: connection
+Multi: boolean
+See-also:
+  - tcp-fastopen
```

Comment:
can you add in the PR description, something like:

```
Attention points:
- This PR replaces #9713.
- Sorry for #13161, I accidentally opened it while the code was not ready yet, my bad.
- Regarding the documentation, we didn't find a good reference (`See-also`). We put `tcp-fastopen` because we had to put something, but we are happy to change.
- It is not clear for us if a test is worth it. According to your comments from #9713, it might not be worth it. We successfully tested it with http://test.multipath-tcp.org:5000 .
```